mirror_builder/wheels.py

```diff
@@ -42,18 +37,22 @@ class BuildEnvironment:
 
     def __init__(self, ctx, parent_dir, build_requirements):
         self._ctx = ctx
-        self._path = parent_dir / f'build-{platform.python_version()}'
+        self.path = parent_dir / f'build-{platform.python_version()}'
         self._build_requirements = build_requirements
         self._createenv()
 
     @property
     def python(self):
-        return (self._path / 'bin/python3').absolute()
+        return (self.path / 'bin/python3').absolute()
 
     def _createenv(self):
+        if self.path.exists():
+            logger.info('reusing build environment in %s', self.path)
```

Comment:
This made a bit nervous at first ... might we end up reusing a build environment between builds, inadvertently?

I think I'd like some safety here - like some way of knowing this is a just-prepared-for-this-exact-build environment that we're "reusing"

Reply:
Filed as #67